feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -93,7 +94,16 @@ export class UpdatePasswordComponentService {
     });
   }
 
-  protected onError(_error: Error): void {
+  protected onError(_error: HttpErrorModel | Error): void {
+    if (
+      _error instanceof HttpErrorModel &&
+      _error.details?.[0].type === 'AccessDeniedError'
+    ) {
+      this.globalMessageService.add(
```

Comment:
I might also ask that we can add a check for this case in a test so that we can block the pipeline in case someone sees this and removes it to avoid regression.